bin/dump-postgres.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+DATE=$(date +"%Y-%m-%d-%H-%M")
+
+DUMPDIR=/home/noiapp/dump
+DESTFILE=$DUMPDIR/dump-$DATE.tar
+LATEST=$DUMPDIR/latest
+ls -t | tail -n +5 | xargs rm --
+docker exec noiapp_noiapp-database_1 bash -c 'pg_dump --create --format t -U noiapp'>$DESTFILE
+rm $LATEST
+ln -s $DESTFILE $LATEST
+ls -t $DUMPDIR | tail -n +4 | xargs rm --
```

Comment:
why do you run it again (see line 8) and having a different `tail` argument?

Reply:
Please can you explain? I don't understand